packages/flutter_android_lifecycle/example/README.md

```diff
@@ -0,0 +1,16 @@
+# flutter_android_lifecycle_example
```

Comment:
optional: I think it's better to delete this completely if we're not going to fill it out.

Reply:
done